test/gui/tst_syncing/test.feature

```diff
@@ -157,7 +157,7 @@ Feature: Syncing files
 
 
     Scenario: Many subfolders can be synced
-        Given user "Alice" has created folder "parent" on the server 
+        Given user "Alice" has created folder "parent" on the server
```

Comment:
you might not need this change in the feature file

Reply:
Yeah, that's by formatter. It won't hurt the test so I think it's ok-ish.